script/watch.js

```diff
@@ -4,24 +4,24 @@
 const blocklist = ['韓國瑜', '韓市長', '韓總', '國瑜', '韓流', '韓粉', '韓導']
 const templateHtml = '<div><h1 style="padding: 30px; text-align: center;">草包已被隱藏！</h1></div>'
 
+const contentFromPosts = document.querySelector('#contentArea')
+const contentFromPages = document.querySelector('#pagelet_timeline_main_column')
+const content = contentFromPosts || contentFromPages;
+
 function removeElems () {
-  blocklist.map(function (text) {
-    $('#pagelet_timeline_main_column div:contains(' + '"' + text + '"' + ')').children('.userContentWrapper').html(templateHtml)
-    $('#pagelet_timeline_main_column p:contains(' + '"' + text + '"' + ')').children('.userContentWrapper').html(templateHtml)
-    $('#pagelet_timeline_main_column span:contains(' + '"' + text + '"' + ')').children('.userContentWrapper').html(templateHtml)
-    $('#pagelet_timeline_main_column a:contains(' + '"' + text + '"' + ')').children('.userContentWrapper').html(templateHtml)
+  const articles = content.querySelectorAll(`div[id][role="article"]`)
+
+  function hasSensitiveWordInBlocklist (article) {
+    return blocklist.some((sensitiveWord) => article.innerHTML.includes(sensitiveWord))
+  }
 
-    $('#contantArea div:contains(' + '"' + text + '"' + ')').children('.userContentWrapper').html(templateHtml)
-    $('#contantArea p:contains(' + '"' + text + '"' + ')').children('.userContentWrapper').html(templateHtml)
-    $('#contantArea span:contains(' + '"' + text + '"' + ')').children('.userContentWrapper').html(templateHtml)
-    $('#contantArea a:contains(' + '"' + text + '"' + ')').children('.userContentWrapper').html(templateHtml)
+  articles.forEach((article) => {
```

Comment:
What about use map and function, instead of arrow function?

Reply:
What's the different between using forEach and map here? We don't actually need the return value.

Reply:
I was read an article that is talk about performance between forEach and Map. and the result is map is faster than forEach (on Node.js). But that is depends on what platform to run the script.

Anyway, we can just use forEach for now. thanks for your replying and PR. I will merge it then.

Reply:
By the way, Could you add `DOMContentLoaded` in listener? thanks.

Reply:
Cool. Thanks for that! I found an article you mentioned about. 
[https://codeburst.io/javascript-map-vs-foreach-f38111822c0f](url)

At first, I guessed that it's some issue about the performance for not to re-invoke the DOMSubtreeModified event, because using forEach will manipulate original array. Instead of using map which will returning the new array.

Reply:
By the way, we can now remove jQuery from the manifest.json file. 😃

Reply:
Yeah, you're totally right, I know that has some performance issue with `DOMSubtreeModified`, but it works really perfect on my laptop (MBPR 15" 2018), so I don't want to spend too much time on that issue; But I know I have to fix that eventually.

By the way, I have been remove jQuery and change back to `forEach`.

Thanks again.

Reply:
I want to let you know a good news. You can install the extension by using Chrome Web Store, and here is it:

https://chrome.google.com/webstore/detail/facebook-%E9%9F%93%E5%9C%8B%E7%91%9C%E9%81%8E%E6%BF%BE%E5%99%A8/aeefpiikngmlmmofdjhhcialiehpjjmf